tests/caasp/stack_update.pm

```diff
@@ -20,18 +20,35 @@ use strict;
 use testapi;
 use lockapi 'mutex_create';
 use caasp 'update_scheduled';
+use caasp 'script_retry';
 use version_utils 'is_caasp';
 
 # Set up ssh to admin node and run update script on all nodes
-sub setup_update_repository {
-    my $repo = update_scheduled;
+sub prepare_to_update {
+    my $repo  = update_scheduled;
+    my $admin = 'admin.openqa.test';
     switch_to 'xterm';
-    assert_script_run "ssh admin.openqa.test './update.sh -s $repo' | tee /dev/$serialdev | grep EXIT_OK", 1200;
+
+    # Add UPDATE repository
+    assert_script_run "ssh $admin './update.sh -s $repo' | tee /dev/$serialdev | grep EXIT_OK", 120;
+
+    # Install packages that should be updated - poo#114205
+    if (is_caasp('qam')) {
+        assert_script_run "ssh $admin './update.sh -i $repo' | tee /dev/$serialdev | grep EXIT_OK", 600;
```

Comment:
`$repo` parameter is not used by update.sh script. The same applies to `-u $repo` below, it should be just `-u`